src/main/java/de/tum/cit/aet/artemis/core/web/AccountResource.java

```diff
@@ -133,46 +139,87 @@ public ResponseEntity<Void> changePassword(@RequestBody PasswordChangeDTO passwo
     }
 
     /**
-     * PUT account/ssh-public-key : sets the ssh public key
+     * GET account/ssh-public-keys : retrieves all SSH keys of a user
+     *
+     * @return the ResponseEntity containing all public SSH keys of a user with status 200 (OK), or with status 400 (Bad Request)
```

Comment:
When do we get status 400 here?

Reply:
Turns out we don't. I removed that part, since always 200 is returned, even when the list is empty.